src/TwilioVideo.ios.js

```diff
@@ -266,24 +273,24 @@ export default class TwilioVideo extends Component {
    * @param  {Boolean} enableNetworkQualityReporting Report network quality of participants
    */
   connect({
-            roomName,
-            accessToken,
-            cameraType = "front",
-            enableAudio = true,
-            enableVideo = true,
-            encodingParameters = null,
-            enableNetworkQualityReporting = false,
-            dominantSpeakerEnabled = false,
-          }) {
+    roomName,
+    accessToken,
+    cameraType = "front",
+    enableAudio = true,
+    enableVideo = true,
+    encodingParameters = null,
+    enableNetworkQualityReporting = false,
+    dominantSpeakerEnabled = false,
+  }) {
     TWVideoModule.connect(
-        accessToken,
-        roomName,
-        enableAudio,
-        enableVideo,
-        encodingParameters,
-        enableNetworkQualityReporting,
-        dominantSpeakerEnabled,
-        cameraType
+      accessToken,
```

Comment:
All of these formatting changes are going to make it more annoying to merge in future changes from the upstream project. I'm guessing your IDE is just auto-formatting the code on save. And if this was a purely independent project I'd encourage that. But since this is a fork of an upstream project that is still actively maintained and receiving changes, I would suggest trying to maintain the existing formatting as much as possible to reduce the chance of future merge conflicts.